config/.env.example

```diff
@@ -1,3 +1,3 @@
-CONFIG_PATH=config/dev/config.toml
+# CONFIG_PATH=config/config.toml
```

Comment:
Was this intentional?

Reply:
Yeah, there's no path for `config/dev` and forgive me if i missed it, but i didnt see any reference to the need for users to create a `dev` folder in the READMEs.

Reply:
Also iiuc, setting `CONFIG_PATH` in `.env` isnt necessary if you're in local dev because `dev.toml` is set by default in `SSIServiceConfig` and if you're not local and want to go to prod, it seems like `docker-compose` is the recommended way to do that, and the `CONFIG_PATH` is set in the  `docker-compose.yml` file, again, rendering the `CONFIG_PATH` in `.env` unnecessary. There are a number of paths between those 2 that users could take (i.e. test.toml) that would require uncommenting and updating the `CONFIG_PATH` in `.env`, so I can add direction about that in the markdown if desired.

Reply:
IIRC, @nitro-neal added `.env` to have an easy way in local dev to change stuff. Perhaps he could add some more color.

Reply:
Actually, as I read it now it makes sense to me.

Reply:
Can I resolve this conversation?